### DISCUSSION TITLE
Displaying image in a label.

### DISCUSSION THREAD

**Discussion by Storme9** [TomSchimansky/CustomTkinter#1759] [General]
I run:

###

image = customtkinter.CTkImage(dark_image="test.jpg")

image_window = customtkinter.CTk()

image_label = customtkinter.CTkLabel(master=image_window, image=image)

###

Line 3 is giving me the error: _tkinter.TclError: image "myimage1" doesn't exist

from the file: "/usr/lib/python3.10/tkinter/__init__.py line 1665

I don't think it is an error my side since the "same" tkinter version is working. Any pointers would be appreciated. 

Thanks

**Comment by avalon60** [comment]
You are using CTkImage incorrectly. 
See the docs:

https://customtkinter.tomschimansky.com/documentation/utility-classes/image

You need to leverage: 

`from PIL import Image`

E.g
```
        from PIL import Image

        upload_image = ctk.CTkImage(light_image=Image.open(images_location / 'upload_lm.png'),
                                    dark_image=Image.open(images_location / 'upload_dm.png'),
                                    size=(40, 40))
```

**Comment by Storme9** [comment]
I made a mistake in my first line of code when describing the problem here BUT NOT in my actual code. The first line should be:

image = customtkinter.CTkImage(dark_image=Image.open("test.jpg"))

This returns the error I described previously.
I had already used leverage. If you do a test and it works then it must be something my end but I see no error in my code.

**Reply by avalon60** [reply]
Can you reproduce the problem with a minimal test case?

**Reply by ghost** [reply]
@avalon60 - Isn't that already available in the OP? I mean apart from calling `.mainloop()` and packing the label widget to the window.

**Comment by ghost** [comment]
Could you post the full traceback?

**Comment by Storme9** [comment]
My minimal test case is producing no errors. Sorry to waste your time gentlemen.

**Reply by ghost** [reply]
No problem :)
